koku/api/report/aws/query_handler.py

```diff
@@ -469,6 +470,22 @@ def __resolve_conditions(condition, alias="t", where=None, values=None):
 
             return res
 
+    def _set_access_filters(self, access, filt, filters):
+        if isinstance(filt, list):
+            for _filt in filt:
+                _filt["operation"] = "in"
+                q_filter = QueryFilter(parameter=access, **_filt)
+                filters.add(q_filter)
+        elif filt["field"] == "organizational_unit__org_unit_path":
+            filt["field"] = "organizational_unit__org_unit_id"
+            filt["operation"] = "in"
+            q_filter = QueryFilter(parameter=access, **filt)
+            filters.add(q_filter)
+        else:
+            filt["operation"] = "in"
+            q_filter = QueryFilter(parameter=access, **filt)
+            filters.add(q_filter)
```

Comment:
The only real difference here between this func and the one in queries.py is the elif. Maybe this could be a bit cleaner?
```suggestion
        if not isinstance(filt, list) and filt["field"] == "organizational_unit__org_unit_path":
            filt["field"] = "organizational_unit__org_unit_id"
       super()._set_access_filters(self, access, filt, filters)
```